src/Core/VersionChecker.cs

```diff
@@ -0,0 +1,49 @@
+using System.Net.Http.Json;
+using System.Reflection;
+using System.Text.Json.Serialization;
+
+namespace Azure.DataApiBuilder.Core;
+
+public static class VersionChecker
+{
+    public static void GetVersions(out string? latestVersion, out string? currentVersion)
```

Comment:
better to return the versions as a value instead of the out references

Reply:
Since it has multiple outputs, aren't `out` parameters a better option than a single return type from the method which would require a complex type or a tuple? Let me know how imperative this request is, and I can change it.

Reply:
I updated the method signature to return bool.